server/backend/api/app/crud/crud_user_role.py

```diff
@@ -157,6 +157,7 @@ def construct_forest_client_role_purpose(
     parent_role_purpose: str, forest_client_number: str
 ):
     LOGGER.debug(f"parent_role_purpose: {parent_role_purpose}")
+    # ???? is there a reason we put a ")" in the client purpose?
     client_purpose = f"{parent_role_purpose} for {forest_client_number})"
```

Comment:
Wonder if there is a reason that we put a ")" in the client purpose? or just a typo?

Reply:
Removed that ")", looks like a typo